### ISSUE TITLE
Cannot Export to VRM in Blender 3.0

### ISSUE THREAD

**Issue by BTHP5** [FlailingFog/KK-Blender-Porter-Pack#135]
I made a new Issue because I had no responses whatsoever, while others had immediate responses.

I'm using the latest master CATS plugin, the 5.10 release of the KKBP plugin, and also Blender 3.0 with the .blend file given within the download of this plug-in.

After fixing the shaders and colours, the next step is to Prepare for Target Export.

However, when I press the VRM option, it gives me an error almost immediately.

Prepping for export...

Combining all objects...

Removing object outline modifier...

Removing duplicate Eye materials...
Using CATS to simplify bones...
Python: Traceback (most recent call last):
File "C:\Users\User\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\KK-Blender-Porter-Pack-master\exporting\selectbones.py", line 73, in execute
bpy.ops.cats_manual.merge_weights()
File "C:\Program Files\Blender Foundation\Blender 3.0\3.0\scripts\modules\bpy\ops.py", line 132, in call
ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Operator bpy.ops.cats_manual.merge_weights.poll() failed, context is incorrect

location: :-1
Error: Python: Traceback (most recent call last):
File "C:\Users\User\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\KK-Blender-Porter-Pack-master\exporting\selectbones.py", line 73, in execute
bpy.ops.cats_manual.merge_weights()
File "C:\Program Files\Blender Foundation\Blender 3.0\3.0\scripts\modules\bpy\ops.py", line 132, in call
ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Operator bpy.ops.cats_manual.merge_weights.poll() failed, context is incorrect

location: :-1

Basically, trying to export to VRM or any other option causes the Plug-in to completely halt at 
“Using CATS to fix armature…” or something alone those lines. 
The plug-in is not actively working when it says this, because I’m able to click buttons and nothing is frozen. 

When trying to fix the bones manually with the CATS plug-in, it gives me an error.

Any help or questions whatsoever would be extremely appreciated.

**Comment by FlailingFog** [comment]
From the error, it looks like the armature wasn't visible when you clicked the button. The plugin should make it visible automatically, but if it didn't that could happen.

**Comment by BTHP5** [comment]
Hi there, I tried unhding the armature, but it didn't seem to help, but I recorded a video of everything I did, I hope it helps!
[Trying to Export the Model](https://youtu.be/MivIIhx3cQE)

**Comment by FlailingFog** [comment]
You're not supposed to open the shader file like that... You need to do it in a blank / default blender file.

**Comment by BTHP5** [comment]
After finally realising that Blender gave no indicaton that the transferring colours step was slowly working in the background, I finally got the colours sorted.

However, when preparing for target export, it gives this error instead 
![image](https://user-images.githubusercontent.com/54001645/160251638-7dd24023-2770-407b-8d6d-777fa3a64c4b.png)

**Comment by BTHP5** [comment]
Is there a way to fix this error? By looking at the log, I'm unsure myself what's causing it

**Comment by BTHP5** [comment]
?

**Comment by FlailingFog** [comment]
Sorry, I don't know... The script is supposed to put the armature into edit mode, select all bones on armature layer 11 and merge the bones using cats, but it looks like nothing is being selected so it fails because there's nothing to merge. It's also failing at line 73 so you can't be using V5.1.0

**Comment by BTHP5** [comment]
Ah I see, I’ll switch to the latest ver and try again :)

**Comment by BTHP5** [comment]
It worked, however I ran into a few bugs:

![image](https://user-images.githubusercontent.com/54001645/160825238-13caeb70-0f2f-486d-a1f9-f87748030a4d.png)
1.When trying to scale the pupils, they wouldn't change no matter what, neither would the eye gradient, or eye highlights

2.When the eyes rotate via eye bones that were created, the pupils clip into the eyewhites.

3. When reapplying the light/normal/dark atlas created by KKBP, the hair, accessories and clothes are transparent, as the plugin cannot find the location of the new textures, so I had to manually enter them into the nodes.

4. The sock textures appear to have random black lines going across them, which are not in the base game's texture.

5. The custom overlay on the shirt is missing

6. Most clothing textures are oversaturated

Thank you so much for all of your help! :)
